packages/runtime/src/widgets/Slider.tsx

```diff
@@ -26,23 +27,31 @@ export type SliderProps = {
   reverse?: boolean;
   dots?: boolean;
   divisions?: number;
+  range?: boolean;
 } & FormInputProps<number | number[]> &
   EnsembleWidgetProps<SliderStyles>;
 
 const SliderWidget: React.FC<SliderProps> = (props) => {
-  const [value, setValue] = useState(props.value);
-  const { values } = useRegisterBindings(
+  const [value, setValue] = useState<number | number[]>();
```

Comment:
all values and initial value should have type `number | [number, number]`. You can alias this for easier reference but it needs to be consistent.

Reply:
@evshi  The `Range` props in the `Slider` allows for more than two nodes, meaning we need to define its value as a number[]. According to the RCSlider documentation and types, the value must be passed as a number[], enabling the creation of multiple nodes within the same slider.

<img width="489" alt="image" src="https://github.com/user-attachments/assets/85088e5f-ac36-4bf0-b5c9-8e8fe935a9c1">

Reply:
Can you provide a link to the documentation? We need to decouple our component interface from the underlying implementation otherwise this will create problems if we need to change the implementation.

Reply:
@evshi Here is the documentation link of `Slider` widget
https://ant.design/components/slider

Reply:
value clearly says `number | [number, number]` - let's focus on the simple case first.